src/containers/StructurePage/StructurePage.jsx

```diff
@@ -235,7 +251,7 @@ export class StructurePage extends React.PureComponent {
                 key={it.id}
```

Comment:
Jeg vet it ikke ble innført nå, men er det mulig å refaktorere til et bedre navn?